cognee/api/client.py

```
@@ -297,90 +308,89 @@ async def add(
-            status_code = 200,
-            content = results,
-        )
+        return results
     except Exception as error:
         return JSONResponse(
             status_code = 409,
             content = {"error": str(error)}
         )
 
-@app.get("/api/v1/settings", response_model=dict)
+from cognee.modules.settings.get_settings import LLMConfig, VectorDBConfig
```

Ruff

Module level import not at top of file

(E402)